ReadMe.md

```diff
@@ -106,6 +109,9 @@ To select | 選択（する） | せんたく |
 To search | 検索（する） | けんさく |
 Object Oriented Programming (OOP) | オブジェクト指向 | オブジェクトしこう |
 Functional Programming (FP) | 関数型プログラミング | かんすうがたプログラミング |
+Artificial Intelligence (AI) | 人工知能 | じんこうちのう |
+Optimization (optimize) | 最適化（する）| さいてきか（する）|
```

Comment:
The nominal form differs in spelling from the verbial form, thus why I added both.

最適：optimum
最適化：optimization
最適化する：to optimize

All have very different ortograph. I think it would benefit this list to make those distinctions clearer whenever applicable (making the list potentially easier to use for non-fluent English speakers).